src/less/form-item.less

```diff
@@ -158,6 +158,10 @@
     line-height: 1;
 }
 
+.form-item--disabled {
+    opacity: .5;
```

Comment:
toto moze byt v niektorych projektoch nechcenne ale spravim minor tam by to nemuselo vadit kedze to nemeni funkcionalitu